src/index.js

```diff
@@ -0,0 +1,17 @@
+import { BrowserRouter } from 'react-router-dom';
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import TodoApp from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <TodoApp />
```

Comment:
- [ OPTIONL] :thinking:  I'm wondering why are you using TodoApp as your component name, kindly change this To Book :wink:

Reply:
Thank @Alfredbis29 I will follow these suggestions.